addons/mail/static/src/components/emoji/emoji.xml

```diff
@@ -2,9 +2,11 @@
 <templates xml:space="preserve">
 
     <t t-name="mail.Emoji" owl="1">
-        <span class="o_Emoji p-1 fs-5 o_cursor_pointer" t-on-click="props.emojiView.emojiListView.onClickEmoji" t-att-title="props.emojiView.emoji.description" t-att-data-source="props.emojiView.emoji.sources[0]" t-att-data-unicode="props.emojiView.emoji.unicode" t-attf-class="{{ className }}" t-ref="root">
+        <t t-if='messaging.emojiRegistry.currentCategory.categoryIcon === "all" || props.emojiView.emoji.emojiCategories.includes(messaging.emojiRegistry.currentCategory)'>
```

Comment:
No special rule for "all".
That's why, in previous commit, you need to `insertAndReplace()` all emojis in "all" category